src/features/wallet/TwoKeyContractModal.tsx

```diff
@@ -88,11 +89,25 @@ const _TwoKeyContractModal: ForwardRefRenderFunction<
         ) : (
           <>
             <p tw="opacity-50 mb-4 text-center">
-              {!hotAddress
-                ? `You are not currently using a two key voting system. To deploy one,
-          provide your cold key address. Click here to learn more about the two
-          key voting system.`
-                : `Your hot wallet address is: ${hotAddress} and your cold wallet address is ${votingAddress}`}
+              {!hotAddress ? (
+                <div>
+                  You are not currently using a two key voting system. To deploy
+                  one, provide your cold key address. Click{" "}
```

Comment:
we should change the copy here and make it clear that this is optional @Tommy1231232

Reply:
@daywiss I don't completely follow. What do you mean change the copy?

Reply:
@Tommy1231232 by "copy" i just mean the text that is shown to the user. I think there was an issue with the old app that it wasnt clear this was optional and people waste gas making a 2 key contract. this doesnt give the user any indication that this is optional, and they probably dont want to do it unless they were told to

Reply:
Gotcha, good idea I'll make a separate CH task for this

Reply:
Good point @daywiss I'll make a separate CH ticket for this